connectionevents.go

```diff
@@ -39,7 +39,7 @@ func (cli *Client) handleStreamError(node *waBinary.Node) {
 	case code == "401" && conflictType == "device_removed":
 		cli.expectDisconnect()
 		cli.Log.Infof("Got device removed stream error, sending LoggedOut event and deleting session")
-		go cli.dispatchEvent(&events.LoggedOut{OnConnect: false, Reason: events.ConnectFailureLoggedOut})
+		go cli.dispatchEvent(&events.LoggedOut{Jid: cli.Store.ID, OnConnect: false, Reason: events.ConnectFailureLoggedOut})
```

Comment:
These are unnecessary and won't be added, the user is expected to remember which client an event came from